earthpy/plot.py

```diff
@@ -185,30 +187,31 @@ def plot_rgb(
 
     Parameters
     ----------
-    arr: numpy ndarray
-        N-dimensional array in rasterio band order (bands, rows, columns)
-    rgb: list
-        Indices of the three bands to be plotted (default = 0,1,2)
-    extent: tuple
-        The extent object that matplotlib expects (left, right, bottom, top)
-    title: string (optional)
-        String representing the title of the plot
-    ax: object
-        The axes object where the ax element should be plotted. Default = none
-    figsize: tuple (optional)
-        The x and y integer dimensions of the output plot if preferred to set.
-    stretch: Boolean
-        If True a linear stretch will be applied
+    arr : numpy array
```

Comment:
Similar comment to above here. Keep default values. And note that if there is a default value the parameter is not optional in the sense it just is fixed until someone changes it. An optional parameter is one that you can use or leave out altogether!